src/main/resources/db/migration/V12__delete_submitted_draft_adjudication_table.sql

```diff
@@ -0,0 +1 @@
+drop table if exists submitted_adjudication_history;
```

Comment:
I take it we are okay with loosing this data in DEV? I guess it means that we won't be able to see reported adjudications that have already been submitted - but I don't think we can do much about that to be honest.